src/stream/src/task/stream_manager.rs

```diff
@@ -539,6 +530,15 @@ impl LocalStreamManagerCore {
         )
         .boxed();
 
+        // If there're multiple stateful executors in this actor, we will wrap it into a subtask.
```

Comment:
What's the purpose of swapping so that WrappedExecutor is inside subtask?

Reply:
https://github.com/risingwavelabs/risingwave/blob/e7b7842826035b645bc4feddf5ef315d0935e35e/src/stream/src/executor/subtask.rs#L57

Reply:
Can we simply fill the identity of `SubtaskRxExecutor` with `{input.identity()} (Rex)` in the wrap function?

Reply:
> What's the purpose of swapping so that WrappedExecutor is inside subtask?
The WrappedExecutor should wrap the actual operator we execute, while the SubtaskRxExecutor should wrap the WrappedExecutor. In the previous implementation, `WrappedExecutor` wraps a `SubtaskRxExecutor`, so that all debug info comes from `SubtaskRxExecutor`.

Reply:
> Can we simply fill the identity of SubtaskRxExecutor with {input.identity()} (Rx) in the wrap function?

I'm not sure. Fake an executor's identity doesn't sound like a good idea?